test/postServiceTest.js

```diff
@@ -0,0 +1,38 @@
+/* global it, describe */
+const sinon = require('sinon');
+const assert = require('assert');
+const Post = require('../models/Post');
+const PostService = require('../services/PostService');
+
+describe('PostService', () => {
+  describe('getOnePost(post_id)', () => {
+    it('should return correct post', () => {
+      const stub = sinon.stub(Post, 'findById').returns(5);
+      const post = PostService.getOnePost(5);
+      assert.equal(post, 5);
+      stub.restore();
+    });
+    it('should be called with correct parameters', () => {
```

Comment:
kind of misleading since "it" refers to `getOnePost()`

Reply:
modified statement within "it"